### DISCUSSION TITLE
⭐🌟STARS🌟⭐

### DISCUSSION THREAD

**Discussion by polekstulod** [Schweinepriester/github-profile-achievements#371] [Q&A] [answered]
Lets exchange stars, Star mine and I Star yours 🥺⭐
https://github.com/polekstulod/foodtrip-api

**Comment by PatrickDSiqueira** [comment, marked as answer]
I still haven't figured out how to get the Galaxy Brain

How does this markup of accepted answers accepted???

Could anyone help me?

**Reply by polekstulod** [reply]
You need to become the marked answer to 2 discussions

**Reply by PatrickDSiqueira** [reply]
Just like you did now?

**Reply by polekstulod** [reply]
Yes, you need one more to get the Galaxy Brain

**Reply by PatrickDSiqueira** [reply]
Ok, more one time, thank you a lot bro

**Comment by PatrickDSiqueira** [comment]
I need only five stars

https://github.com/PatrickDSiqueira/BuscadorCEP

**Reply by polekstulod** [reply]
done , thank you!

**Reply by PatrickDSiqueira** [reply]
thank you bro

**Reply by WHAHA-HA** [reply]
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aziz0x48** [reply]
@PatrickDSiqueira @WHAHA-HA done https://github.com/c99tn/xSMTP

**Reply by PatrickDSiqueira** [reply]
> Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

done too

**Reply by PatrickDSiqueira** [reply]
> @PatrickDSiqueira @WHAHA-HA done https://github.com/c99tn/xSMTP

Done in your repository,

**Reply by ArielMAJ** [reply]
@PatrickDSiqueira I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by WHAHA-HA** [comment]
@polekstulod
Done, https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by polekstulod** [reply]
thank you!

**Comment by polekstulod** [comment]
up

**Comment by scarabaeus** [comment]
@polekstulod Starred! https://github.com/scarabaeus/scarabaeus

**Reply by polekstulod** [reply]
done, thank you!

**Reply by aziz0x48** [reply]
@scarabaeus done https://github.com/c99tn/xSMTP

**Reply by miladkhalighi** [reply]
Done 
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by scarabaeus** [reply]
Done!

**Reply by Preetiraj3697** [reply]
@scarabaeus done give me back https://github.com/Preetiraj3697/Sheela_Ecommerce_Website

**Comment by miladkhalighi** [comment]
Done
https://github.com/miladkhalighi/Food-Delivery-Flutter-app

**Reply by aziz0x48** [reply]
@miladkhalighi done https://github.com/c99tn/xSMTP

**Reply by polekstulod** [reply]
@miladkhalighi done, thank you!

**Reply by Preetiraj3697** [reply]
@miladkhalighi done give me back star https://github.com/Preetiraj3697/Sheela_Ecommerce_Website

**Reply by Preetiraj3697** [reply]
@c99tn done give me back star https://github.com/Preetiraj3697/Sheela_Ecommerce_Website

**Reply by ArielMAJ** [reply]
@miladkhalighi I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by 41pha1** [comment]
Done
https://github.com/41pha1/MIDI-Converter

**Comment by xLevitate** [comment]


**Reply by ArielMAJ** [reply]
@xLevitate I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by karthikeya-Git** [comment]
Done 👍
Just Need few more - https://github.com/abhinit21/abhinit21

**Reply by ArielMAJ** [reply]
@karthikeya-Git I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by bstummer** [comment]
Star and I'll star your repo back
https://github.com/bstummer/openskill.lua

**Reply by Preetiraj3697** [reply]
@bstummer Done give me a back star
https://github.com/Preetiraj3697/Sheela_Ecommerce_Website

**Reply by ArielMAJ** [reply]
@bstummer I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Rantoryu** [reply]
@bstummer I gave You a star :)
If You can give me star too I would appreciate it.
https://github.com/Rantoryu/Personal_Website

**Comment by Preetiraj3697** [comment]
@polekstulod  Done give me back to star
https://github.com/Preetiraj3697/Sheela_Ecommerce_Website

**Reply by ArielMAJ** [reply]
@Preetiraj3697 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by DushanSenadheera** [comment]
Hello! can anyone help me to achieve starstruck achievement by giving some starts to this repo https://github.com/DushanSenadheera/Algorithms-and-Data-Structures

**Reply by Preetiraj3697** [reply]
Done 
https://github.com/Preetiraj3697/Sheela_Ecommerce_Website  please give me back.

**Reply by ArielMAJ** [reply]
@DushanSenadheera I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Rantoryu** [comment]
@DushanSenadheera @Preetiraj3697 I've starred your repos. Please, star mine too.
https://github.com/Rantoryu/Personal_Website
